utils/aplay/aplay.c

```diff
@@ -1148,18 +1198,26 @@ int main(int argc, char *argv[]) {
 		for (i = 0; i < ba_addrs_count; i++, tmp += 19)
 			ba2str(&ba_addrs[i], stpcpy(tmp, ", "));
 
+		char index_str[sizeof(",NNN")];
+		snprintf(index_str, sizeof(index_str), ",%u", mixer_elem_index % 1000);
```

Comment:
Why `mixer_elem_index` is wrapped here? If it needs to be constrained to [0, 1000], is should be done in the "getopt" logic.